### DISCUSSION TITLE
Cannot use Alpine in v3

### DISCUSSION THREAD

**Discussion by psjavi** [livewire/livewire#7171] [1. Help]
Hi, I've got in my layout.blade the following config:
```

<!DOCTYPE html>
<html>
<head>
    @livewireStyles
</head>
<body>
@yield('content')

@livewireScriptConfig
<script src="{{ mix('js/app.js') }}"></script> 
</body>
</html>
```
On the other hand I've got a component with
```

    <div x-data>
        <button @click="console.log($code)"></button>

        <h2 x-text="$wire.code"></h2>
    </div>
```

I'm getting the following errors:

![image](https://github.com/livewire/livewire/assets/4563033/618b5e35-846c-4255-9501-2c272b2dc838)

What I'm doing wrong?

**Comment by Zakarialabib** [comment]
Are u using webpack🙄 try making use of vite its way more better than mix , 
 check your app.js maybe u missed the line that uses livewire and alpinejs at the end livewire.start

**Comment by psjavi** [comment]
I've discovered that If I remove @livewireStyles and @livewireScriptConfig all works correctly and the JS error disappear. I don't understand why but It works.

**Comment by joshhanley** [comment]
@psjavi there is an issue with bundling Alpine and Livewire using Mix as Mix doesn't support bundling into ES2017. So I would upgrade to using Vite and then it should work happily. Shift has a free upgrade tool https://laravelshift.com/convert-laravel-mix-to-vite

Hope this helps!
